cmd/ocm-backplane/config/set.go

```diff
@@ -0,0 +1,68 @@
+package config
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/openshift/backplane-cli/pkg/cli/config"
+)
+
+var configFlags struct {
+	url      string
+	proxyURL string
+}
+
+func newSetCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:          "set",
+		Short:        "Set Backplane CLI configuration variables",
+		Example:      "ocm backplane config set --proxy-url <proxy> --url <url>",
+		SilenceUsage: true,
+		RunE:         setConfig,
+	}
+
+	cmd.Flags().StringVar(
+		&configFlags.proxyURL,
+		ProxyURLConfigVar,
+		"",
+		"Squid proxy URL",
+	)
+
+	cmd.Flags().StringVar(
+		&configFlags.url,
+		URLConfigVar,
+		"",
+		"Backplane API URL",
+	)
+
+	return cmd
+}
+
+func setConfig(cmd *cobra.Command, args []string) error {
+	// Retrieve default Backplane CLI config path, $HOME/.config/backplane/config.json
+	configPath, err := config.GetConfigFilePath()
+	if err != nil {
+		return err
+	}
+
+	if configFlags.proxyURL == "" {
```

Comment:
```
$ ocm backplane2 config set --url https://www.example.com
ERRO[0000] proxy-url should not be empty                
$ ocm backplane2 config set --proxy-url http://www.example.com:3128
ERRO[0000] url should not be empty                      
```

Should be able to set them separately?

Reply:
We cannot unfortunately, `viper` overwrites the entire config file while saving, hence all the config parameters should be provided.

Reply:
How about we read the file when setting, if the file exists we can load the existing value and update the one from the parameter?

Reply:
I am not sure which way is better approach, but I feel all the parameters should be set at the same time looks a bit inconvenient.
And I think with the other PR in the queue, there will be more variables added to the configfile. like this https://github.com/openshift/backplane-cli/pull/105/files#diff-66bf83773b78df966f05c4b0ea1985e249c5fc46ae55f942a0adcc431e904d3cR337

Reply:
If the variable list is going to increase, I'll port to allow individual variables instead